src/pet.js

```diff
@@ -0,0 +1,76 @@
+const minFitness = 0;
+const minAcceptableFitness = 3;
+const maxFitness = 10;
+const minHunger = 0;
+const minAcceptableHunger = 5;
+const maxHunger = 10;
+const maxAge = 30;
```

Comment:
generally, constants representing magic numbers are written in screaming snake case, e.g.
`const MIN_FITNESS = 0`